R/salesforce.R

```diff
@@ -78,7 +78,12 @@ querySalesforceDataWithQuery <- function(server = NULL, username, password, secu
 #' @param securityToken - (optional) security token to login
 #' @param table - The table you want to get data
 #' @param columns - list of columns that you want to get data from the table.
-querySalesforceDataFromTable <- function(server = NULL, username, password, securityToken = NULL, table = NULL, columns = NULL, guessType = TRUE, conditions = NULL, limit = NULL){
+#' @param dataType - list of column data types for the above columns parameter
+#' @param guessType - flag to tell that dataType should be detected. If FALSE is passed, all the data types become character.
+#' @param conditions - (optional) SOQL where conditions.
+#' @param limit - number of rows to return.
+#' @param logicalOperator (optional) - required if the conditions are set. By default it uses "AND" logical operator to construction SOQL where clause.
```

Comment:
Check the actual values of "AND", "OR".

Reply:
Updated to lower case since Desktop sends "and" and "or" as lower case.